master/static/srv/proto_get_trials_plus.sql

```diff
@@ -42,6 +42,8 @@ trial_validations AS (
     ) IS NOT NULL
 ),
 best_validation AS (
+  -- is the window function needed here?
```

Comment:
I believe it'd still be a nested query with group by. we could benchmark it though.